Documentation/6.0.2/README.rst

```diff
@@ -0,0 +1,1518 @@
+
+    ::
+
+        cp /usr/share/openstack-tripleo-heat-templates/roles/* /home/stack/nuage-tripleo-heat-templates/roles/
+
+
+**For an AVRS integration please follow below steps as well**:
+
+User can have Single or Mutli-Roles for AVRS nodes.
+
+    **For single-role AVRS deployment**, use the `create_compute_avrs_role.sh <../../nuage-tripleo-heat-templates/scripts/create_roles/create_compute_avrs_role.sh>`_ to create a role file called ``compute-avrs-role.yaml``.
+
+    Run using
+
+    ::
+
+         cd /home/stack/nuage-tripleo-heat-templates/scripts/create_roles
+
+        ./create_compute_avrs_role.sh
+
+
+    Above command will create a new ``ComputeAvrs``  role for your deployment, and compare it with sample `compute-avrs-role-sample.yaml <../../nuage-tripleo-heat-templates/templates/compute-avrs-role-sample.yaml>`_ .
+
+    **For mutli-role AVRS deployment**, we have automated `script <../../nuage-tripleo-heat-templates/scripts/create_roles/create_compute_avrs_multirole.sh>`_ to create ComputeAvrsSingle & ComputeAvrsDual role. You can edit these files with your requirements to create new roles.
+    You can read more about usage of roles at https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/director_installation_and_usage/index#sect-Generate_Architecture_Specific_Roles
+
+    Run using
+
+    ::
+
+        cd /home/stack/nuage-tripleo-heat-templates/scripts/create_roles
+        ./create_compute_avrs_multirole.sh
+
+
+
+
+Phase 5: Generate a CMS ID for the OpenStack installation.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Cloud Management System (CMS) ID needs to be generated to configure your OpenStack installation with the VSD installation.
+
+Go to /home/stack/nuage-ospdirector/generate-cms-id for the files and script to generate the CMS ID, and follow the instructions in README.md.
+
+The CMS ID is displayed in the output, and a copy of it is stored in a file called cms_id.txt in the same folder.
+
+Add the CMS ID to the /home/stack/nuage-tripleo-heat-templates/environments/neutron-nuage-config.yaml template file for the ``NeutronNuageCMSId`` parameter.
+
+
+Phase 6: Check the Ironic node status to ensure that the Ironic nodes have been successfully created.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following commands.
+
+1. Run the following command. The results should show the *Provisioning State* status as *available* and the *Maintenance* status as *False*.
+
+::
+
+    openstack baremetal node list
+
+
+2. If profiles are being set for a specific placement in the deployment, run the following command. The results should show the *Provisioning State* status as *available* and the *Current Profile* status as *control* or *compute*.
+
+::
+
+    openstack overcloud profiles list
+
+
+Phase 7: Create the Heat templates.
```

Comment:
Revise text as follows.

From:
Create the Heat templates.

To: 
Create the Heat Templates